### PR TITLE
Pysimplegui

### PR DESCRIPTION
**Pull request by agcopenhaver**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pysimplegui/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ styledName }}-{{ version }}.tar.gz"
+  sha256: "19fdf1bf59af9d07dfee3cfcbe393a25b95746a973b07583d33cb468c6adb5fe"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/pysimplegui/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/pysimplegui/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - PySimpleGUI
+
+about:
+  home: "https://github.com/PySimpleGUI/PySimpleGUI"
+  license: "GNU Lesser General Public v3 or later (LGPLv3+)"
```
Comment (marcelotrevisani) [inline recipes/pysimplegui/meta.yaml]:
```suggestion
  license: "LGPL-3.0"
```

**Comment by marcelotrevisani** [comment]
I suggested some improvements, I think after that it should be good to go

**Comment by agcopenhaver** [comment]
Thanks @marcelotrevisani

**Comment by agcopenhaver** [comment]
we good to merge @marcelotrevisani ?

File: recipes/pysimplegui/yum_requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+mesa-libGL
```
Comment (pierrsc) [inline recipes/pysimplegui/yum_requirements.txt]: You should confirm that you need mesa-libGL for this package.

Comment (agcopenhaver) [reply recipes/pysimplegui/yum_requirements.txt]: Prior to adding to as a yum-requirement, I got the following error during testing:

ImportError: libX11.so.6: cannot open shared object file

for linux builds. After searching for a solution, I found the following thread:

https://github.com/conda-forge/itkwidgets-feedstock/pull/6

which resolved the issue.

**Pull request merged**